.travis.yml

```diff
@@ -21,12 +21,12 @@ env:
   matrix:
     - TEST="clang-format, ament_lint"
     - ROS_DISTRO=dashing
+      BEFORE_SCRIPT=apt install liboctomap-dev
```

Comment:
We need to add octomap to rosdep as a system dependency...

Reply:
Of course, if it is available